### PR TITLE
[PW-4729] Add region based cloud api endpoints

### PR DESCRIPTION
**Pull request by AlexandrosMor**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: Adyen.Test/BaseTest.cs [hunk 1]
```diff
@@ -320,7 +320,7 @@ protected Client CreateAsyncMockTestClientApiKeyBasedRequest(string fileName)
         /// <returns>IClient implementation</returns>
         protected Client CreateMockTestClientPosCloudApiRequest(string fileName)
         {
-            var config = new Config { Endpoint = ClientConfig.CloudApiEndPointTest };
+            var config = new Config { Endpoint = ClientConfig.CloudApiEndPointTestSync };
```
Comment (Aleffio) [inline Adyen.Test/BaseTest.cs:323]: shouldn't `Config.CloudApiEndPoint` be used instead?
`Config.Endpoint` is the one for classic integration(pal)?
```suggestion
            var config = new Config { CloudApiEndPoint = ClientConfig.CloudApiEndPointTestSync };

```

File: Adyen/Constants/ClientConfig.cs [hunk 2]
```diff
@@ -32,8 +32,13 @@ public class ClientConfig
         public static string HppLive = "https://live.adyen.com/hpp";
         public static string CheckoutEndpointTest = "https://checkout-test.adyen.com";
         public static string CheckoutEndpointLiveSuffix = "-checkout-live.adyenpayments.com/checkout";
+        //Test cloud api endpoints
         public static string CloudApiEndPointTest = "https://terminal-api-test.adyen.com";
-        public static string CloudApiEndPointLive = "https://terminal-api-live.adyen.com";
+        //Live cloud api endpoints
+        public static string CloudApiEndPointEULive = "https://terminal-api-live.adyen.com";
+        public static string CloudApiEndPointAULive = "https://terminal-api-live-au.adyen.com";
+        public static string CloudApiEndPointUSLiveSync = "https://terminal-api-live-us.adyen.com";
```
Comment (Aleffio) [inline Adyen/Constants/ClientConfig.cs:40]:
```suggestion
        public static string CloudApiEndPointUSLive = "https://terminal-api-live-us.adyen.com";
```

File: README.md [hunk 3]
```diff
@@ -72,6 +72,19 @@ dotnet build
...
 
+### Using the Cloud API for post
+In order to submit POS request with Cloud API you need to initialize the client with the Endpoints that it is closer to your region. The Endpoints are available as contacts in [ClientConfig](https://github.com/Adyen/adyen-dotnet-api-library/blob/develop/Adyen/Constants/ClientConfig.cs#L35)
+For more information please read our [documentation](https://docs.adyen.com/point-of-sale/terminal-api-fundamentals#cloud)
+~~~~ csharp
+//Example for EU based Endpoint Syncronous
+var config = new Config
+  {
+    XApiKey = "Your merchant XAPI key",
+    CloudApiEndPoint = ClientConfig.CloudApiEndPointEULiveSync
```
Comment (Aleffio) [inline README.md:83]:
```suggestion
    CloudApiEndPoint = ClientConfig.CloudApiEndPointEULive
```

**Review by Aleffio — APPROVED** [review]

**Review by deepu105 — APPROVED** [review]
Would this be a breaking change?

**Pull request merged**